.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/backend"
+  schedule:
+    interval: daily
```

Comment:
The docker schedules can be set to weekly as they are not used frequently

Reply:
Addressed it.